examples/lib/defaultHtmlStepUi.js

```diff
@@ -125,23 +130,24 @@ function DefaultHtmlStepUi(_sequencer, options) {
                          </div>";
         step.ui.querySelector("div.details").appendChild(div);
       }
-
-      $(step.ui.querySelector("div.details")).append(
-        '<div class="cal collapse in"><p><button type="submit" class="btn btn-default btn-save" disabled = "true" >Apply</button><span> Press apply to see changes</span></p></div>'
-      );
-
-      
+      $(step.ui.querySelector("div.panel-footer")).append(
+        '<div class="cal collapse in"><p><button type="submit" class="btn btn-sm btn-default btn-save" disabled = "true" >Apply</button><span> Press apply to see changes</span></p></div>'
+      );      
+      $(step.ui.querySelector("div.panel-footer")).append(
+        '<button class="pull-right btn btn-sm insert-step" >\
+      <i class="fa fa-plus"></i> Insert Module\
+      </button>'
+      );  
     }
 
     if (step.name != "load-image") {
       step.ui
-        .querySelector("div.details")
+        .querySelector("div.panel-heading")
         .appendChild(
```

Comment:
Ah, here i think we want `prepend` instead of `appendChild` and it'll insert in the header, with `pull-right`. https://api.jquery.com/prepend/

Reply:
@jywarren i fixed it with css .

Reply:
Hmm, i would generally prefer to try to use just standard Bootstrap CSS, if possible -- it's more maintainable, you know? Can you push your latest changes so we can see? Thank you!